runners/genlayer-py-std/src/genlayer/gl/nondet/web.py

```
@@ -0,0 +1,160 @@
+		return None
+	if isinstance(data, str):
+		return data.encode('utf-8')
+	return data
+
+
+@_lazy_api
+def get(
+	url: str,
+	*,
+	headers: dict[str, str | bytes] = {},
+) -> Lazy[Response]:
```

Ruff (0.11.9)

`Lazy` may be undefined, or defined from star imports

(F405)